README.md

```diff
@@ -310,6 +311,13 @@ spec:
 
 Traces will be written to the stdout logs of the Gatekeeper controller.
 
+
+If there is an error in the Rego in the ConstraintTemplate, there are cases where it is still created via `kubectl apply -f constraint-template.yaml`.
+
+When applying the Constraint using `kubectl apply -f constraint.yaml` with a ConstraintTemplate that contains incorrect Rego, and error will occur: `error: unable to  recognize "constraint.yaml": no matches for kind "[NAME_OF_CONSTRAINT]" in version "constraints.gatekeeper.sh/v1alpha1"`.
+
+To fix the error, it is useful to view the logs, by running the command `kubectl logs -n gatekeeper-system gatekeeper-controller-manager-0 manager`.
```

Comment:
In master we have a fix that allows the user to see build errors when in the `status` field of the constraint template.

We should probably cut a new release.

@ritazh Can we cut a new release? I'm unsure of the process there.

@jessica-dl Can you add in the info about running `kubectl get` on the CT, and we can block merging this PR on a new release?

Reply:
@maxsmythe Pushed a commit with the updated info.

Reply:
sure I can cut v3.0.2 today